docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -133,6 +133,7 @@ Now, we have the following known optional plugins.
 * Plugin of Spring Cloud Gateway 2.1.x in optional plugin folder. Please only active this plugin when you install agent in Spring Gateway.
 * Plugin of [Play Framework](https://www.playframework.com/) 2.6+ (JDK 1.8 required & Scala 2.12/2.13) in optional plugin folder. Please only active this plugin when you install agent in [Play Framework](https://www.playframework.com/). 
 * Plugin of Spring Transaction in optional plugin folder. The reason of being optional plugin is, many local span are generated, which also spend more CPU, memory and network.
+* Plugin of Spring Webflux 5.x in optional plugin folder. Agent requires JDK 1.8+,  Please make sure you use JDK1.8, when using this Agent
```

Comment:
```suggestion
* Plugin of Spring Webflux 5.x in the optional plugin folder. Because the plugin requires JDK 1.8+.
```

~~Please make sure you use JDK1.8, when using this Agent~~. We don't need this, once user is using Webflux, they have JDK 1.8+ for sure.

Reply:
ok